app/helpers/check_answers_helper.rb

```diff
@@ -37,6 +48,10 @@ def items
     }.compact.flatten
   end
 
+  def skip_question?(question)
+    question.in?(SKIPPABLE_QUESTIONS) && session[question].blank?
```

Comment:
May not be the right forum for this but I'm going to ask questions given my newity! Does a user have to explicitly skip or just not enter any info and click continue?

Reply:
They don't see these questions at all. If they answer "no" to one of the preceding questions, they jump right over them. 

e.g. https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/master/app/controllers/coronavirus_form/hotel_rooms_controller.rb#L17-L23

If they say "no" to any hotel rooms, they jump straight to the transport question and are not asked how many hotel rooms they have.